shell/install.sh

```diff
@@ -147,12 +147,12 @@ download() {
 
 bin_sha512() {
   case "$OPAM_BIN" in
-    opam-2.0.1-arm64-linux)    echo "5231ea96f19328c42d3f207de76121c8c84887810087dd21f9703ec619045f5a2323e20b15e6c69fd95c784e5438514088418146d6f9f61fca391d4507724c94";;
-    opam-2.0.1-armhf-linux)    echo "5c7f8c38c0bbad503046fb7a443c6425cf122658c6f1d8dd21a980d840da19baf9d318659dbe27a5c05022a480648e128a1df00d30a32538351c1fee9a63215d";;
-    opam-2.0.1-i686-linux)     echo "10e93a82e2ff31415cd0e92fc452fc9c303968e50aa4062585fe02cc1ab60c440ef805e911bbba9eca7ff14b47443c71156ecb69d77f2b9454be25f113d81010";;
-    opam-2.0.1-x86_64-darwin)  echo "f49b4658e37932b3f1541e70d885b0a53e9da973080554bebc1f090f68597172a5b4686ef61b59fcb728e1e83b6800fe171ddcf4fcceeacd49d7a4bc9e22bc70";;
```

Comment:
opam-2.0.1-x86_64-*macos*
(change to `macos` also above)
e111633b4bcac56c7a813fef141080ef0a9701f7945c59d123d7667b92b4fb9c080ff703e9ceee8451aee4ee9ad8ea0ed3a63c13911a9cc394eb304fa81d0935